db/migrate/20221122214935_add_user_type_to_users.rb

```diff
@@ -0,0 +1,7 @@
+class AddUserTypeToUsers < ActiveRecord::Migration[6.1]
+  # This user_type column will eventually allow us to have more than
+  # two user types. See https://github.com/sciencehistory/scihist_digicoll/issues/1948
+  def change
+    add_column :users, :user_type, :string, default: 'editor'
```

Comment:
you sure you want to make the default "editor"? Where does that come from?  I guess we can change the default later, let's not forget to if needed!

Reply:
Of course. I figure when you create a new user you will probably not want to make then an admin by default.